documentation/milestones/individual_milestones/static_site.md

```diff
@@ -0,0 +1,154 @@
+
+By sharing this information in a publicly accessible format and investing early in the infrastructure used to host it, this site aims to demonstrate the following value propositions:
+
+- Raising awareness about the Grants Equity project, its priorities, and ongoing workstreams
+- Establishing beta.grants.gov as the primary location that stakeholders can visit for project updates and previews of deliverables from each of the workstreams
+- Proving the succesful completion of technical milestones that enable faster development without sacrificing code quality or security
+- Delivering an early win that both internal and external stakeholders can rally around, which helps build momentum and enthusiam for the project
+
+### User stories
+
+- As a **full-time HHS staff member**, I want:
+  - TODO
+- As a **grantor**, I want:
+  - TODO
+- As a **member of an HHS contractor team**, I want:
+  - TODO
+- As an **open source contributor**, I want:
+  - TODO
+- As a **prospective grant applicant**, I want:
+  - TODO
+
+## Technical description
+
+### {Sub-deliverable}
+
+TODO
+
+### Definition of done
+
+- [ ] The following infrastructure requirements are satisfied:
+  - [ ] The code needed to build and deploy the site is merged to `main`
+  - [ ] The site is built and hosted with the tools selected in the [Front-end Planning milestone](https://github.com/HHS/grants-equity/issues/49)
+  - [ ] All code quality checks set up in the [Developer Tools milestone](https://github.com/HHS/grants-equity/issues/50) are passing
+  - [ ] The resources required to deploy and host the site are provisioned programmatically using the framework established in the [Infrastructure-as-Code milestone](https://github.com/HHS/grants-equity/issues/123)
+  - [ ] Code changes are deployed using the CI/CD pipeline set up in [the Front-end CI/CD milestone](https://github.com/HHS/grants-equity/issues/58)
+- [ ] The following user experience (UX) requirements are satisfied:
+  - [ ] Anyone can access a live version of the site at beta.grants.gov
+  - [ ] The site adopts the UI principles and framework established in the [Foundational UI milestone](https://github.com/HHS/grants-equity/issues/60)
+  - [ ] The site is 508 compliant and satisfies the other guidelines outlined in the Accessibility Planning milestone
+- [ ] The following content requirements are satisfied:
+  - [ ] The site contains information about both the NOFO Transformation and Grants.gov Modernization workstreams
+  - [ ] The content on the site has been been reviewed and approved by the relevant stakeholders within each workstream
+  - [ ] The site also links to external resources related to the project, including:
+    - [ ] The legacy grants.gov site
+    - [ ] The open source repository
+    - [ ] The open source wiki
+    - [ ] The open source chat
```

Comment:
Note: These aren't currently in the draft content.

Reply:
But it does include a way to provide free-form feedback.

Reply:
I modified the line before these resources are listed to say: 

```
The site also links to external resources related to the project (if they are available), including:
```

I do think these would be important to include if they are available. But if there's another reason that we wouldn't want to link to these resources, we can strike them from this list.